shacl12-test-suite/tests/core/node/memberShape-001.ttl

```diff
@@ -0,0 +1,94 @@
+@prefix dash: <http://datashapes.org/dash#> .
+@prefix ex: <http://datashapes.org/sh/tests/core/node/memberShape-001.test#> .
+@prefix mf: <http://www.w3.org/2001/sw/DataAccess/tests/test-manifest#> .
+@prefix owl: <http://www.w3.org/2002/07/owl#> .
+@prefix rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#> .
+@prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#> .
+@prefix sh: <http://www.w3.org/ns/shacl#> .
+@prefix sht: <http://www.w3.org/ns/shacl-test#> .
+@prefix xsd: <http://www.w3.org/2001/XMLSchema#> .
+
+ex:PersonListShape
+  rdf:type sh:NodeShape ;
+  rdfs:label "Person List Shape" ;
+  sh:targetClass ex:PersonList ;
+  sh:memberShape [
+      a sh:NodeShape ;
+      sh:class ex:Person ;
+    ] ;
+.
+
+ex:Person
+  rdf:type rdfs:Class ;
+  rdf:type sh:NodeShape ;
+  rdfs:label "Person" ;
+.
+
+ex:Animal
+  rdf:type rdfs:Class ;
+  rdf:type sh:NodeShape ;
+  rdfs:label "Animal" ;
+.
+
+ex:Alice
+  rdf:type ex:Person ;
+  rdfs:label "Alice" ;
+.
+
+ex:Bob
+  rdf:type ex:Person ;
+  rdfs:label "Bob" ;
+.
+
+ex:Rex
+  rdf:type ex:Animal ;
+  rdfs:label "Rex" ;
+.
+
+# Valid person list with all Person instances
+ex:validList
+  rdf:type ex:PersonList ;
+  rdf:value (
+      ex:Alice
+      ex:Bob
+    ) ;
+  rdfs:label "Valid Person List" ;
+.
```

Comment:
Shouldn't this be spelled:

```turtle
ex:validList
  rdf:type ex:PersonList ;
  rdf:first ex:Alice ;
  rdf:rest (
    ex:Bob
  ) ;
  rdfs:label "Valid Person List" ;
.
```

`rdf:value` is a property, but has no meaning, per [5.4.3 or RDF 1.2 today](https://www.w3.org/TR/2025/WD-rdf12-schema-20250224/#ch_value).

I forget if you would also need to assert `ex:PersonList rdfs:subClassOf rdf:List`.